CHANGES.md

```diff
@@ -720,14 +720,14 @@
 * sync: main -> 1.5.x by @TorstenDittmann in https://github.com/appwrite/appwrite/pull/7705
 * Sync main 1.5.x by @TorstenDittmann in https://github.com/appwrite/appwrite/pull/7727
 * Updated header by @DylanG-64 in https://github.com/appwrite/appwrite/pull/7728
-* chore: update exector, runtimes by @loks0n in https://github.com/appwrite/appwrite/pull/7729
+* chore: update executor, runtimes by @loks0n in https://github.com/appwrite/appwrite/pull/7729
```

Comment:
Some projects don't like changing changelogs. I'm happy to drop if it's a problem